Library/SettingsAccountCellType.swift

```diff
@@ -48,8 +43,6 @@ public enum SettingsAccountCellType: SettingsCellTypeProtocol, CaseIterable {
 
   public var detailTextColor: UIColor {
     switch self {
-    case .currency:
```

Comment:
Any reason for removing this? I believe we still want the currency text to display in green instead of of the default grey color

Reply:
Not according to [this](https://github.com/kickstarter/ios-oss/pull/568#issuecomment-460772246) comment.